src/xclim/indicators/atmos/_precip.py

```diff
@@ -815,3 +816,15 @@ class HrPrecip(Hourly):
     cell_methods="time: sum over days",
     compute=indices.water_cycle_intensity,
 )
+
+antecedent_precipitation_index = Indicator(
```

Comment:
```suggestion
antecedent_precipitation_index = Daily(
```
Subclassing from `Daily` will add a check that ensures the input data is Daily. From the docstring, I gathered that's something you'd want ?

Reply:
Sounds right, yes.